saltproc/app.py

```diff
@@ -165,22 +170,27 @@ def read_main_input(main_inp_file):
 
         # Saltproc settings
         process_file = str((input_path /
-                              j['proc_input_file']).resolve())
+                              input_parameters['proc_input_file']).resolve())
         dot_file = str((
             input_path /
-            j['dot_input_file']).resolve())
-        output_path = j['output_path']
-        num_depsteps = j['num_depsteps']
+            input_parameters['dot_input_file']).resolve())
+        output_path = input_parameters['output_path']
+        n_depletion_steps = input_parameters['n_depletion_steps']
 
         # Global output path
         output_path = (input_path / output_path)
-        j['output_path'] = output_path.resolve()
+        input_parameters['output_path'] = output_path.resolve()
+
+        # Create output directoy if it doesn't exist
+        if not Path(input_parameters['output_path']).exists():
+            Path(input_parameters['output_path']).mkdir(parents=True)
 
         # Class settings
-        depcode_input = j['depcode']
-        simulation_input = j['simulation']
-        reactor_input = j['reactor']
+        depcode_input = input_parameters['depcode']
+        simulation_input = input_parameters['simulation']
+        reactor_input = input_parameters['reactor']
 
+        depcode_input['codename'] = depcode_input['codename'].lower()
```

Comment:
Is this supposed to be
```suggestion
        codename = depcode_input['codename'].lower()
```
?